src/main/kotlin/nl/avisi/structurizr/site/generatr/site/SiteGenerator.kt

```diff
@@ -70,6 +71,28 @@ private fun copyAssets(assetsDir: File, exportDir: File) {
     assetsDir.copyRecursively(exportDir, overwrite = true)
 }
 
+private fun generateStyle(context: GeneratorContext, exportDir: File) {
+    val configuration = context.workspace.views.configuration.properties
+    val primary = configuration.getOrDefault("structurizr.style.colors.primary", "#333333")
+    val secondary = configuration.getOrDefault("structurizr.style.colors.secondary", "#cccccc")
+
+    val file = File(exportDir, "style-branding.css")
+    val content = """
+        .has-site-branding {
+            background-color: $primary!important;
+            color: $secondary!important;
+        }
+        .has-site-branding::after {
+            border-color: $secondary!important;
+        }
+        .menu-list a.is-active {
+            background-color: $primary;
```

Comment:
Should `color: $secondary;` be added here as well, like the header?

Reply:
Yes, missed that one because the default was the color I had chosen for testing, thanks.